build.py

```diff
@@ -664,6 +671,10 @@ def __init__(self, args):
         # MayaUsd Location
         self.mayaUsdLocation = (os.path.abspath(args.mayausd_location).replace("\\","/")
                                 if args.mayausd_location else None)
+
+        # MtoA Location
+        self.mtoaLocation = (os.path.abspath(args.mtoa_location)
+                                if args.mtoa_location else None)
```

Comment:
Add the replace here (like it is for mayaUsdLocation) above. Then these are sent to the BuildAndInstall and will have the / so no replace needed there.